src/dist/replication/lib/replica_2pc.cpp

```diff
@@ -44,6 +44,19 @@ void replica::on_client_write(dsn::message_ex *request, bool ignore_throttling)
         return;
     }
 
+    if (dsn_unlikely(_stub->_max_allowed_write_size &&
+                     request->body_size() > _stub->_max_allowed_write_size)) {
+        dwarn_replica("client from {} write request body size exceed threshold = {}, gpid = "
+                      "({}.{}), it will be reject!",
+                      request->header->from_address.to_string(),
+                      _stub->_max_allowed_write_size,
+                      get_gpid().get_app_id(),
```

Comment:
get_gpid().to_string()

Reply:
I found `dwarn_replica` will auto add `gpid@ip` info, such as `[2.1@127.0.0.1]`, so it can be del